lib/load-logos.js

```diff
@@ -2,13 +2,14 @@ var fs = require('fs');
 var path = require('path');
 
 var loadLogos = function() {
+  // Cache svg logos from disk in base64 string
   var logos = {};
-  var logoFiles = fs.readdirSync(path.join(__dirname, '..', 'logo'));
+  var logoDir = path.join(__dirname, '..', 'logo');
+  var logoFiles = fs.readdirSync(logoDir);
   logoFiles.forEach(function(filename) {
     if (filename[0] === '.') { return; }
     // filename is eg, github.svg
-    var svg = fs.readFileSync(
-      path.join(__dirname, '..', 'logo', filename)).toString();
+    var svg = fs.readFileSync(logoDir + '/' + filename).toString();
```

Comment:
On my Windows it works okay, so I don't see why it is needed here. It probably matters if you have to call some external Windows program and pass the path there, which is not the case.